subsys/nrf_security/src/zephyr/CMakeLists.txt

```diff
@@ -87,7 +87,12 @@ if(TARGET mbedx509)
   target_link_libraries(mbedtls_external INTERFACE mbedx509)
 endif()
 
+# Link with legacy include library to get includes for Mbed TLS _ALT files
+if(TARGET mbedcrypto_legacy_includes)
+  zephyr_link_libraries(mbedcrypto_legacy_includes)
+endif()
```

Comment:
`mbedcrypto_legacy_includes` is created in `src/drivers/legacy`, whereas this Zephyr code is directly under `src/zephyr`.

If `mbedcrypto_legacy_includes` belongs under drivers, then I think this code should be placed in corresponding `src/drivers/zephyr/CMakeLists.txt`.

Note, that will probably require the lines:
https://github.com/frkv/fw-nrfconnect-nrf/blob/958475715d0018e5a6721d7d4fcf3c9a7b7a608f/subsys/nrf_security/src/drivers/CMakeLists.txt#L31-L34

to be moved before:
https://github.com/frkv/fw-nrfconnect-nrf/blob/958475715d0018e5a6721d7d4fcf3c9a7b7a608f/subsys/nrf_security/src/drivers/CMakeLists.txt#L14

and/or the `  add_subdirectory(zephyr)` to be moved outside of the safeguard.

Reply:
@frkv Will you revise the changes to address it? Should I wait?

Reply:
I don't want to add more risk with adding additional overhaul. This started failing since we did a change and I'm doing the smallest possible fixups retroactively to make it look like it was before the change